runtime/gc_base/ObjectAccessBarrier.cpp

```diff
@@ -1546,7 +1571,19 @@ MM_ObjectAccessBarrier::copyObjectFields(J9VMThread *vmThread, J9Class *objectCl
 		while (offset < limit) {
 			UDATA srcAddress = (UDATA)srcObject + srcOffset + offset;
 			UDATA destAddress = (UDATA)destObject + destOffset + offset;
-			if (sizeof(uint32_t) == referenceSize) {
+			bool copyQword = false;
```

Comment:
Please choose a name other than `Qword` - it's intel-specific and extra confusing given this code is dealing with Q types. `copy64Bits` would be better.

Reply:
Changed to `copy64Bits`.